lints/lint_old_root_ca_rsa_mod_less_than_2048_bits.go

```diff
@@ -26,7 +26,11 @@ func (l *rootCaModSize) CheckApplies(c *x509.Certificate) bool {
 }
 
 func (l *rootCaModSize) RunTest(c *x509.Certificate) (ResultStruct, error) {
-	mod := c.PublicKey.(*rsa.PublicKey).N
+	key, found := c.PublicKey.(*rsa.PublicKey)
```

Comment:
I'll change it to ok.
If you read the description parts in the lints. The former is "In a validity period beginning on or before 31 dec 2010, root CA certificates using RSA public key algorithm must  have 2048 bits of modulus". While the latter is "in validity period beginning after 31 Dec 2010, all certificates using RSA public key algorithm must have 2048 bits of modulus".